packages/lab/src/components/Wizard/Wizard.tsx

```diff
@@ -6,7 +6,7 @@ import React, { useCallback, useEffect, useMemo, useState } from "react";
 import { HvWizardContainer } from "./WizardContainer";
 import { HvWizardTitle, HvWizardTitleProps } from "./WizardTitle";
 import { HvWizardContent } from "./WizardContent";
-import HvWizardContext, { HvWizardTabs } from "./WizardContext";
+import { HvWizardContext, HvWizardTabs } from "./WizardContext";
```

Comment:
Same as above, for all the `./` imports

```suggestion
import { HvWizardContext, HvWizardTabs } from ".";
```

Reply:
Yep, you're right. I'll change it 👍